JMMServer/Commands/TvDB/CommandRequest_TvDBDownloadImages.cs

```diff
@@ -37,7 +37,7 @@ public string PrettyDescription
 		public CommandRequest_TvDBDownloadImages()
 		{
 		}
-
+        /*

```

Comment:
I'm guessing you have commented this out because it is has 0 references?
The reason it has 0 references is because the class isn't actually used anywhere at the moment, but if it was you need this constructor